### ISSUE TITLE
CocoaLumberjack.podspec loss the setting of dependency with  'CoreGraphics' and 'UIKit'

### ISSUE THREAD

**Issue by ph661** [CocoaLumberjack/CocoaLumberjack#1137]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

CocoaLumberjack.podspec   settings of **Core**  lost  `ss.frameworks  = 'CoreGraphics', 'UIKit'`

When I set  `CLANG_ENABLE_MODULES` is `NO` in my project, build failure!

```
s.subspec 'Core' do |ss|
  ss.source_files         = 'Sources/CocoaLumberjack/**/*.{h,m}'
  ss.private_header_files = 'Sources/CocoaLumberjack/DD*Internal.{h}'
  ss.frameworks  = 'CoreGraphics', 'UIKit'
end
```

**Comment by ffried** [comment]
See comment / discussion in #1138

**Comment by bpoplauschi** [comment]
I think we can close this issue, per https://github.com/CocoaLumberjack/CocoaLumberjack/pull/1139#pullrequestreview-410897173 
Also, this issue seems a duplicate of #1071
